circuit_knitting/cutting/__init__.py

```diff
@@ -60,6 +61,17 @@
     qpd.decompose_qpd_instructions
     qpd.qpdbasis_from_instruction
 
+Automatic Cut Finding
+=====================
```

Comment:
Also, maybe this should be a subheader, since it should logically be fall under the "Circuit Cutting" header above.
```suggestion
Automatic Cut Finding
~~~~~~~~~~~~~~~~~~~~~
```